pandas/plotting/_matplotlib/core.py

```diff
@@ -153,8 +158,8 @@ def __init__(
 
         self.grid = grid
         self.legend = legend
-        self.legend_handles = []
-        self.legend_labels = []
+        self.legend_handles: List = []
```

Comment:
What error does this solve?

Reply:
this solves mypy complaint, exactly the same as in https://github.com/pandas-dev/pandas/pull/28373